api/v1beta2/foundationdb_env_variables.go

```diff
@@ -0,0 +1,26 @@
+/*
+ * foundationdb_env_variables.go
+ *
+ * This source file is part of the FoundationDB open source project
+ *
+ * Copyright 2022-2024 Apple Inc. and the FoundationDB project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v1beta2
+
+const (
+	// EnvNamePublicIP defines the FDB_PUBLIC_IP environment variable name.
+	EnvNamePublicIP = "FDB_PUBLIC_IP"
```

Comment:
Created: https://github.com/FoundationDB/fdb-kubernetes-operator/issues/2029 for adding more env variables here.